Library/Homebrew/dev-cmd/dispatch-build-bottle.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+
+require "cli/parser"
+require "utils/github"
+
+module Homebrew
+  module_function
+
+  def macos_versions
+    ["10.13", "10.14", "10.15"]
+  end
+
+  def dispatch_build_bottle_args
+    Homebrew::CLI::Parser.new do
+      usage_banner <<~EOS
+        `dispatch-build-bottle` [<options>] <formula> [<formula> ...]
+
+        Build bottles for these formulae with GitHub Actions.
+      EOS
+      flag   "--tap=",
+             description: "Target tap repository (default: `homebrew/core`)."
+      flag   "--issue=",
+             description: "If specified, post a comment to this issue number if the job fails."
+      flag   "--macos=",
+             description: "Version of macOS the bottle should be built for."
+      flag   "--workflow=",
+             description: "Dispatch specified workflow (default: `dispatch-build-bottle.yml`)."
+      switch "--upload",
+             description: "Upload built bottles to Bintray."
+    end
+  end
+
+  def dispatch_build_bottle
+    args = dispatch_build_bottle_args.parse
+
+    raise FormulaUnspecifiedError if args.named.empty?
+
+    odie "Must specify --macos option" unless args.macos
+
+    unless macos_versions.include?(args.macos)
+      odie "Unsupported macOS version specified, currently supported: #{macos_versions}"
+    end
```

Comment:
```suggestion
    begin
      MacOS::Version.from_symbol(args.macos.to_sym)
    rescue MacOSVersionError
      MacOS::Version.new(args.macos)
    end
```

which will allow both `catalina` or `10.15` and fail if it's an unknown version.